app/assets/js/main.js

```diff
@@ -0,0 +1,142 @@
+/**
+ * Created by student on 17.11.16.
+ */
+
+$(document).ready(function () {
+
+    $(".site-spinner").fadeOut("slow", function () {
+        $(this).remove();
+    });
+
+
+    var change = $('.change');
+
+    change.each(function(){
+        var imgSrc = $(this).children().attr('src');
+        $(this).children("img").remove();
+        $(this).css('background-image', 'url("'+imgSrc+'")');
+    });
+
+
+    $('#owl-main').owlCarousel({
+        items: 1,
+        margin: 0,
+        dots: true,
+        loop: true,
+        autoplay: true,
+        autoplayTimeout: 5000,
+        nav: true,
+        navText: ['<span><img src="assets/img/arrow_left.png" width="50px" height="50px"></span>', '<span><img src="assets/img/arrow_right.png" width="50px" height="50px"></span>']
+
+    });
+
+
+    $("#owl-work").owlCarousel({
+
+        items: 1,
+        margin: 0,
+        nav: false,
+        dots: true,
+        loop: true,
+        autoplay: true,
+        autoplayTimeout: 7000
+
+        // "singleItem:true" is a shortcut for:
```

Comment:
You shouldn't commit commented code. If this code is not needed then just remove it.